lib/utils/win-releases.js

```diff
@@ -84,13 +84,13 @@ function parseRELEASES(content) {
             if (filenameParts.length >= 4) {
                 let offset = 0;
                 if (isAlpha) {
-                    offset = 1000;
+                    offset = 100000;
```

Comment:
Maybe these could change to
CHANNEL_MAGNITUDE
CHANNEL_MAGNITUDE * 2
CHANNEL_MAGNITUDE * 3
etc?

Reply:
Good call. Done.